src/lib/helpers/config.ts

```diff
@@ -15,6 +15,10 @@ export const ConfigSchema = z.object({
         enable_metrics: z.boolean().default(
             Deno.env.get("SERVER_ENABLE_METRICS") === "true" || false,
         ),
+        track_unknown_innertube_errors: z.boolean().default(
```

Comment:
Why introducing a way to disable or enable those metrics?

One can simply just extract what he wants using prometheus and not all the metrics if needed.

Reply:
Yeah, seems right to just track it from the start. It's just a single counter.